ccl/serverccl: TestAdminAPIDataDistributionPartitioning failed

ccl/serverccl.TestAdminAPIDataDistributionPartitioning [failed](https://tanzanite.cluster.engflow.com/invocation/19b917c6-82af-48d7-8d27-359d1af53e0b) on master @ [e747f6e6857a19d6048cb184b0f55c52cb8a6390](https://github.com/cockroachdb/cockroach/commits/e747f6e6857a19d6048cb184b0f55c52cb8a6390):


```
=== RUN   TestAdminAPIDataDistributionPartitioning
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestAdminAPIDataDistributionPartitioning2490985862
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:93: cluster virtualization disabled due to issue: #106897 (expected label: C-bug)

pkg/ccl/serverccl/admin_test.go:93: (TestAdminAPIDataDistributionPartitioning)
	NOTICE: .GetAuthenticatedHTTPClient() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().GetAuthenticatedHTTPClient() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    admin_test.go:94: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/serverccl/admin_test.go:94
        	Error:      	Received unexpected error:
        	            	Get "https://127.0.0.1:34941/_admin/v1/data_distribution": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
        	Test:       	TestAdminAPIDataDistributionPartitioning
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestAdminAPIDataDistributionPartitioning2490985862
--- FAIL: TestAdminAPIDataDistributionPartitioning (79.06s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cluster-observability
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAdminAPIDataDistributionPartitioning.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34308

--- a/pkg/ccl/serverccl/admin_test.go
+++ b/pkg/ccl/serverccl/admin_test.go
@@ -19,6 +19,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/server/serverpb"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -45,6 +46,8 @@ func getAdminJSONProtoWithAdminOption(
 func TestAdminAPIDataDistributionPartitioning(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+
+	skip.UnderRace(t, "this test creates a 3 node cluster; too resource intensive.")
 
 	// TODO(clust-obs): This test should work with just a single node,
 	// i.e. using serverutils.StartServer` instead of

